modules/sandboxed-containers-selecting-nodes.adoc

```diff
@@ -3,5 +3,31 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-selecting-nodes_{context}"]
+= Selecting nodes for {sandboxed-containers-first}
 
-= Selecting nodes for OpenShift sandboxed containers
+You can selectively install the Kata runtime on specific workers.
+
+.Prerequisites
+
+* You have {product-title} 4.8 installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Identify the labels you want to use for selecting your nodes. For this example, select nodes with labels to be chosen as candidates to run on your {sandboxed-containers-first} workloads. If the nodes exist, they are selected.
+.. To apply a label to a node, run the following command: `oc label node <worker_node_name> custom-kata=test`. This labels your worker node with the `custom-kata` label that has a value of `test`.
+. Edit `KataConfig` custom resource (CR) `oc edit kataconfig` to add a label selector.
```

Comment:
Would putting the goal first work here? e.g.

> To add a label selector, edit the `KataConfig` custom resource (CR):

Could then put `oc edit kataconfig` on its own line, with the YAML codeblock under it as "example `KataConfig` CR" or similar.